.github/workflows/golangci-lint.yml

```diff
@@ -17,3 +17,4 @@ jobs:
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
           version: v1.35
+          args: --timeout=120s 
```

Comment:
Makes, sense, can we have both for a long-time test? I think this solves it for now and a longer timeout on the run does not hurt anything IMHO.

Reply:
We can merge this but it will only hide the real issue. I would not do it until we have more info about the real issue behind it, *then* enable a higher timeout if that is the actual problem.

Reply:
Ok I will remove this again.